curp/src/server/curp_node.rs

```diff
@@ -798,7 +886,8 @@ impl<C: 'static + Command, RC: RoleChange + 'static> CurpNode<C, RC> {
             ae.prev_log_term,
             ae.entries,
             ae.leader_commit,
-        )?;
+        )
+        .map_err(|err| SendAEError::EncodeDecode(err.to_string()))?;
```

Comment:
Why not impl `From<bincode::Error>` for `SendAEError`?

Reply:
Done